session4/src/com/itschool/homework/Challenge2.java

```diff
@@ -0,0 +1,25 @@
+package com.itschool.homework;
+
+public class Challenge2 {
+
+    public static void main(String[] args) {
+
+        int a = 1;
+        int b = 2;
+        //increment or decrement
+
+        System.out.println(a++);
+        System.out.println(++a);
+        System.out.println(++b);
+        System.out.println(b++);
```

Comment:
here the value changed at 'b++' is never used, meaning the output is still 3.  Can you tell why? And how can we print the incremented value?

Reply:
I wanted to see how the output is not changed by using the 'b++' value, as a post-increment operator. In order to print the incremented value, I wrote line 13, value '++b', using the pre-increment operator.